docs/developer/concepts/duck-typing.md

```diff
@@ -2,17 +2,29 @@
 
 Knative enables [loose coupling](https://en.wikipedia.org/wiki/Loose_coupling) of its components by using [duck typing](https://en.wikipedia.org/wiki/Duck_typing).
 
-Duck typing means that the compatibility of a resource for use in a Knative system is determined by certain properties being present that can be used to identify the resource control plane shape and behaviors. These properties are based on a set of common definitions for different types of resources, called duck types.
+Duck typing means that the compatibility of a resource for use in a Knative system is determined by
+certain properties that are used to identify the resource control plane shape and behaviors.
+These properties are based on a set of common definitions for different types of resources, called
+duck types.
+<!-- What are the duck types? The properties? The common definitions? The types of resources? -->
 
-If a resource has the same fields in the same schema locations as the common definition specifies, and the same control or data plane behaviors as the common definition specifies, Knative can use that resource as if it is the generic duck type, without specific knowledge about the resource type. Some resources may choose to opt-in to multiple duck types.
+Knative can use a resource as if it is the generic duck type, without specific knowledge about the
+resource type, if:
+
+* The resource has the same fields in the same schema locations as the common definition specifies
+* The same control or data plane behaviors as the common definition specifies
+
+Some resources can opt in to multiple duck types.
 
 <!-- TODO: point to Discovery ClusterDuckType documentation. -->
 
-A fundamental use of duck typing in Knative is the use of object references in
-resource _specs_ to point to another resource. The definition of the object
-containing the reference prescribes the expected duck type of the resource being referenced.
+A fundamental use of duck typing in Knative is using object references in resource _specs_ to point
+to other resources.
+The definition of the object containing the reference prescribes the expected duck type of the
+resource being referenced.
 
```

Comment:
Maybe add a heading here to show where the introduction ends and the example begins?

Reply:
Good idea; I've done so.